updates links

ioduok opened the pull request:
to docs-internal.rockarch.org for systems-info-sheet links

ioduok committed:
updates links to docs-internal.rockarch.org systems-info-sheet links

p-galligan commented:
This line does not actually link to the systems info sheet (I'm not sure it should, but if not, it shouldn't be a link).

https://github.com/RockefellerArchiveCenter/archivematica-local/blob/cb24cf9fab7545d5c60a8ca81c99e5c2b645a4e4/index.md?plain=1#L13

p-galligan commented:
I'd add instructions on where to look at logs on the Storage Service here:

https://github.com/RockefellerArchiveCenter/archivematica-local/blob/cb24cf9fab7545d5c60a8ca81c99e5c2b645a4e4/administration.md?plain=1#L68

p-galligan commented on updating.md:
I would remove the `may` from `may include` in line 57. It _does_ include that.

p-galligan commented on updating.md:
I'd say that we also want to include performing the Fornax integration test in the final section (9) as well. Since we want to test integrations in production too

ioduok committed:
minor content change to updating.md

change made in section ### 8. Update Integrated Applications (Archivematica Product Owner)

ioduok committed:
Update section 9 updating.md

add instructions about turning Fornax Start Transfer service off for testing
